File: Java/PC.java
/* membuat class */ 
class PC{

	/* deklarasi atribut private */
	private	String name;
	private Processor p_obj;
	private Disk d_obj;
	private RAM r_obj;
	private int totalPrice = 0;

	/* constructor */
	public PC(){}

	public PC(String name, Processor p, Disk d, RAM r){
		this.name = name;
		this.p_obj = p;
		this.d_obj = d;
		this.r_obj = r;
	}

	/* setter dan getter untuk atribut name */
	public void setName(String name){ 
		this.name = name;
	}

	public String getName(){
		return this.name;
	}

	/* setter dan getter untuk class Processor */
	public void setProcessor(Processor p){ 
		this.p_obj = p;
	}

	public Processor getProcessor(){
		return this.p_obj;
	}

	/* setter dan getter untuk class Disk */
	public void setDisk(Disk d){ 
		this.d_obj = d;
	}

	public Disk getDisk(){
		return this.d_obj;
	}

	/* setter dan getter untuk class RAM */
	public void setRAM(RAM r){ 
		this.r_obj = r;
	}

	public RAM getRAM(){
		return this.r_obj;
	}

	/* method untuk menghitung totalPrice */
	public int getTotalPrice(){
		this.totalPrice = this.p_obj.getPrice() + this.d_obj.getPrice() + this.r_obj.getPrice();
		return this.totalPrice;
	}

	/* method untuk mencetak output atribut dari class */ 
	public void printPC(){
		System.out.println("\nNama Produk : " + this.name);
		System.out.println("---------------------------------------");
		this.p_obj.printProcessor();
		this.d_obj.printDisk();
		this.r_obj.printRAM();
		System.out.println("Harga       : Rp" + this.getTotalPrice());
	}
}

File: Python/PC.py
# import file 
from Processor import Processor
from Disk import Disk
from RAM import RAM

# membuat class
class PC():

	# deklarasi atribut private
	__name = ""
	__p_obj = Processor()
	__d_obj = Disk()
	__r_obj = RAM() 
	__totalPrice = 0

	# constructor 
	def __init__(self):
		self.__name = ""
		self.__p_obj = Processor()
		self.__d_obj = Disk()
		self.__r_obj = RAM()

	# setter dan getter untuk atribut name
	def setName(self, name):
		self.__name = name

	def getName(self):
		return self.__name

	# setter dan getter untuk class Processor 
	def setProcessor(self, __p_obj):
		self.__p_obj = __p_obj

	def getProcessor(self):
		return self.__p_obj

	# setter dan getter untuk class Disk
	def setDisk(self, __d_obj):
		self.__d_obj = __d_obj

	def getDisk(self):
		return self.__d_obj

	# setter dan getter untuk class RAM
	def setRAM(self, __r_obj):
		self.__r_obj = __r_obj

	def getRAM(self):
		return self.__r_obj

	# method untuk menghitung totalPrice 
	def getTotalPrice(self):
		self.__totalPrice = self.__p_obj.getPrice() + self.__d_obj.getPrice() + self.__r_obj.getPrice()
		return self.__totalPrice

	# method untuk mencetak output atribut dari class 
	def printPC(self):
		print("Nama Produk : " + self.getName())
		print("---------------------------------------")
		self.__p_obj.printProcessor()
		self.__d_obj.printDisk()
		self.__r_obj.printRAM()
		print("Harga       : Rp{}".format(self.getTotalPrice()))